api/can_api_v2_definition.py

```diff
@@ -237,6 +237,9 @@ class Actuals(base_model.APIBaseModel):
         None,
         description="Number of individuals who are fully vaccinated and have received a booster (or additional) dose.",
     )
+    vaccinationsBivalentDose: Optional[int] = pydantic.Field(
+        None, description="Number of individuals who have received a bivalent vaccine dose.",
```

Comment:
Sorry, I should have flagged this in the scraper PR too, but as we discussed at the kickoff this morning I think we should probably come up with a better name than "bivalent dose".  Most likely we'll have a new booster sometime later this year that could be bivalent or trivalent or quadvalent or whatever.  So to be more future-proof, we should probably do "Fall 2022 booster" or "Fall 2022 bivalent booster" or something.

Reply:
Ah, yeah

I tend to agree--this name doesn't seem great. I'm not 100% sure what a good/future-proof alternative might be--maybe `vaccinationsFirstBivalentDose` / `vaccinations_first_bivalent_dose`?

Reply:
`vaccinationsFall2022Booster` / `vaccinationsFall2022BivalentBooster` both sound good to me, too